grails-app/domain/com/mini/asaas/customer/Customer.groovy

```diff
@@ -1,11 +1,14 @@
 package com.mini.asaas.customer
 
 import com.mini.asaas.base.BasePerson
+import com.mini.asaas.payer.Payer
 
 class Customer extends BasePerson {
 
     CompanyType companyType
 
+    static hasMany = [payers: Payer]
```

Comment:
A longo prazo isso gera um baita problema, pq um cliente pode ter N payer e quando for instanciar o customer vai puxar todos os payers, recomendo não utilizar hasMany